nni/nas/space/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from .frozen import *
+# from .graph import *
+from .metrics import *
+# from .mutator import *
```

Comment:
why not directly remove the commented lines?

Reply:
These files have already been done. They are just in another PR. Hide them for now to avoid import error.